playbooks/installer/ospd/overcloud/pre.yml

```diff
@@ -146,7 +146,7 @@
         shell: "source ~/stackrc; openstack baremetal introspection bulk start"
 
       - name: get subnet ID to update neutron's DNS server
-        shell: "source ~/stackrc; neutron subnet-list | grep {{ installer.undercloud.config.network_cidr }} | awk '{print $2;}'"
+        shell: "source ~/stackrc; neutron subnet-list | grep {{ installer.undercloud.network.network_cidr }} | awk '{print $2;}'"
```

Comment:
These parameters (and any parameter that have "installer.undercloud.config") are related to undercloud.conf file.
The idea was to be able to take a conf file from everyone (for BM deployments we will need to get a undercloud.conf file from a user) and insert it to "installer.undercloud.config" so later on in the playbook, we will use the content here to update the undercloud.conf file.
Because of this, I don't think it's corrent to rename this to "network".